bnelson777/id.ly
# IL-127: Store/Retrieve Private Keys in Sensitive Info Package

- Private keys and public keys are now added to sensitive storage in create_card. Public keys are also stored in card object.
- Decrypt message in lockbox has been modified to get private keys from sensitive storage based on matching public key.
- Clear data function has been updated to remove keys from sensitive storage. The function clears an arbitrary number of keys for simplicity and because it is ultimately a dummy function.

Testing:
1. Clear data and reload the app.
2. Go to wallet and create a new card.
3. Send a message from the new card and select email when it appears in lockbox.
4. Open the link from email and observe that it is added to your messages.
